docs/api-and-integration/metadata/jira.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Jira Integration
+description: Configuring Octopus integration to Jira.
+---
+
+In addition to being able to [track metadata and work item](index.md) information through your CI/CD pipeline, Octopus can also integrate with Jira to provide progress information related to the work items during deployments.
+
+The integration to Jira is additional to the work-item tracking built in to Octopus. The configuration described below assumes you already have the work-item tracking enabled in your pipeline and packages.
+
+## Deployment Events
+
+To set the scene, the end goal of the integration is to provide Jira with updates on the progress of work-items through the pipeline. It will display information similar to the following when it receives these updates for the work-items:
+
+![Jira Deployments](jira-deployment.png)
+
+As a note, _Development_ here refers to an environment type, not an environment name. The environment name in Octopus in this example was actually Azure-Dev. See below for more detail on where this becomes important.
+
+An important thing to note is that the link in the image above will link to the deployment in Octopus. In order for Octopus to feed this Url to Jira **you must have the Server Uri configured on the Configuration/Nodes page**.
+
+## Octopus Connect App and the Jira Extension
+
+Something worth pointing out at this point is that the Jira functionality we're demonstrating here is only available in the SaaS version of Jira. There isn't a SaaS requirement on the Octopus side of the integration though, so you can use either a self-hosted or Cloud version of Octopus.
+
+With that in mind, the first step in the integration is to add the Octopus Connect App in the Jira marketplace.
+
+***TODO: add a screenshot of the marketplace tile***
+
+To complete the installation of the app from the marketplace you must configure some security related values. This requires aligning a couple of pieces of information from the page that appears in Jira:
+
+![Jira ConnectApp configuration](jira-app-config.png)
+
+And from the **Configuration / Jira Issue Tracker** page in Octopus:
+
+![Octopus Jira Issue Tracker configuration](octo-jira-config.png)
+
+You'll probably want both of these open in separate browser windows/tabs to complete the configuration. On the Jira side you need to copy the _Octopus Installation Id_ from the Octopus page, and on the Octopus side you'll need to copy your Jira instance's *Base Url* and the *Octopus Secret* that appears on the Connect App configuration page to use as the *Jira Connect App Password*.
```

Comment:
```suggestion
You'll probably want both of these open in separate browser windows/tabs to complete the configuration. On the Jira side you need to copy the `Octopus Installation Id` from the Octopus page, and on the Octopus side you'll need to copy your Jira instance's `Base Url` and the `Octopus Secret` that appears on the Connect App configuration page to use as the `Jira Connect App Password`.
```

maybe? Not entirely convinced thats not an abuse of formatting

Reply:
I've swapped both code/back-ticks and italics out for bold.